app/packages/operators/src/operators.ts

```diff
@@ -484,6 +485,16 @@ export async function executeOperator(operatorURI, params: any = {}) {
   queue.add(request);
 }
 
+export async function validateOperatorInputs(
```

Comment:
should we use this in useOperatorPrompt as well?

Reply:
I was suprised not to see `Operator#validate()` if we normalize it should probably be there. I think we can do that in another PR though.